### DISCUSSION TITLE
[Help] zk: error: no editor set in config

### DISCUSSION THREAD

**Discussion by wtLau** [zk-org/zk#327] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

I am getting this error. How can i solve it?

So I've installed zk via "Build from scratch" method. Ran `make`. No compile error. Ran `zk new --title "xxx". This error was thrown.

I am use NVim with AstroVim distro, not sure how those two would connect...

Any help would be appreciated! I really want to use this software. Thx alot🙏🏽

**Comment by basilioss** [comment, marked as answer]
Try to set `EDITOR` or `VISUAL` environment variable, for example `export EDITOR=nvim`.  There is also an option to change the editor in the configuration file `.zk/config.toml` under the `[tool]` section.

**Reply by wtLau** [reply]
`export EDITOR=nvim` works and solve my problem. Thx

I don't know why but I dont have the zk config folder ` .zk/config.toml` in my `~` directory

**Reply by basilioss** [reply]
It should be in your notebook directory created with `zk init`. Have a look at https://github.com/mickael-menu/zk/blob/main/docs/getting-started.md and https://github.com/mickael-menu/zk/blob/main/docs/config.md.

**Reply by wtLau** [reply]
Got it! Thx :D

**Reply by mickael-menu** [reply]
You can also set it globally in `~/.config/zk/config.toml`. But setting your `EDITOR` env variable is the "proper" way to fix this.
